### ISSUE TITLE
Bump to Node 20

### ISSUE THREAD

**Issue by andriiholubenko** [Azure/setup-kubectl#94]
changes already merged into main branch but not released. do you have any news?

**Comment by davidgamero** [comment]
yes the new release workflow is being added this week which will unblock this

**Comment by andriiholubenko** [comment]
thank you

**Comment by pacoguzman** [comment]
Hi there,

I think there is an issue with the new release workflow. As you can see after the last merge onto [main](https://github.com/Azure/setup-kubectl/actions/runs/7718484552) the release workflow failed

```sh
[Invalid workflow file: .github/workflows/release-pr.yml#L16](https://github.com/Azure/setup-kubectl/actions/runs/7718484552/workflow)
error parsing called workflow
".github/workflows/release-pr.yml"
-> "Azure/action-release-workflows/.github/workflows/release_js_project.yaml@e4a1a0385530d6861c9a9b7262058ad33b10c769"
: failed to fetch workflow: workflow was not found.
```

I'm not totally sure how that works but if `e4a1a0385530d6861c9a9b7262058ad33b10c769` is a commit sha on the other repository, that commit does not exist -> https://github.com/Azure/action-release-workflows/commit/e4a1a0385530d6861c9a9b7262058ad33b10c769

I hope this help to debug the issue.

**Comment by perkrlsn** [comment]
@davidgamero any news on this? As @pacoguzman pointed out it seems like the release has failed.

I created a [PR](https://github.com/Azure/setup-kubectl/pull/97) to set the version on `action-release-workflows` to the latest release.

**Comment by perkrlsn** [comment]
Thanks @davidgamero, it works like charm :+1:  The node 16 deprecation notice is gone 🥳

**Comment by davidgamero** [comment]
i am glad to hear that it is working

closing this issue for now, we can reopen if a further issue is discovered
